site/src/examples/highcharts-theme/ColumnChart.tsx

```diff
@@ -0,0 +1,79 @@
+import { Switch } from "@salt-ds/core";
+import { useChart } from "@salt-ds/highcharts-theme";
+import { clsx } from "clsx";
+import Highcharts, { type Options } from "highcharts";
+import accessibility from "highcharts/modules/accessibility";
+import HighchartsReact from "highcharts-react-official";
+import { useRef, useState } from "react";
+import styles from "./index.module.css";
+
+accessibility(Highcharts);
+
+const options: Options = {
+  chart: {
+    type: "column",
+  },
+  title: {
+    text: "Regional revenue by product",
+  },
+  accessibility: {
+    description:
+      "A column chart comparing revenue across regions. Each category shows a single column representing the total value for that region.",
+  },
```

Comment:
```suggestion
 point: {
      valueDescriptionFormat:
        "{index}. {xDescription}, {value} million dollars.",
    },
  },
```
This should help screen readers announce the correct amount rather than just the number

Reply:
Makes sense